packages/wrangler/src/index.ts

```diff
@@ -567,20 +572,16 @@ export function createCLIParser(argv: string[]) {
 	wrangler.command(
 		"deployments",
 		"🚢 Displays the 10 most recent deployments for a worker",
-		(yargs) => {
+		(yargs) =>
 			yargs
 				.option("name", {
 					describe: "The name of your worker",
 					type: "string",
 				})
-				.epilogue(deploymentsWarning);
-		},
-		async (deploymentsYargs: ArgumentsCamelCase<{ name: string }>) => {
+				.epilogue(deploymentsWarning),
+		async (deploymentsYargs) => {
```

Comment:
...ish. The correct type for this is `ArgumentsCamelCase<CommonYargsOptions & { name: string }>`, so `CommonYargsOptions` wasn't included before. Since `yargs` can completely infer that, it seems best to let the inference work it's magic.

Reply:
I can never argue with letting "inference work its magic"